docker/webui/docker-compose-example.yaml

```diff
@@ -0,0 +1,82 @@
+version: '3'
+services:
+  data:
+    image: openqa_data
+    restart: always
+    volumes:
+      - ./data/factory:/data/factory
+      - ./data/tests:/data/tests
+  backend:
+    image: openqa-webui
+    ports:
+      - "9527:9527"
+      - "9529:9529"
```

Comment:
The port 9528 is missing. That would be the port of the live handler (which falls into the same category). I also don't see any other trace of the live handler so it seems missing here. (The architecture diagram covers all services, theirs default ports and connections: https://open.qa/docs/images/architecture.svg)